classes/helper/login_helper.php

```diff
@@ -519,16 +549,25 @@ public function is_username_set_in_auth_global_vars() {
      *
      * @return bool
      */
-    public function is_user_already_loggedin() {
+    public function is_user_already_loggedin()
+    {
         return isloggedin() and !isguestuser();
     }
 
     /**
      * Clear the username from
      */
-    public function clear_username_in_auth_global_vars() {
+    public function clear_username_in_auth_global_vars()
+    {
         global $frm;
```

Comment:
No need to declared global again once declared in the script, unless within function call.